drivers/usb/device/usb_dc_mcux.c

```diff
@@ -910,6 +910,9 @@ static void usb_mcux_thread_main(void *arg1, void *arg2, void *arg3)
 		case kUSB_DeviceNotifyResume:
 			dev_state.status_cb(USB_DC_RESUME, NULL);
 			break;
+		case kUSB_DeviceNotifySOF:
```

Comment:
Where is `kUSB_DeviceNotifySOF`. This is causing CI failures.

Reply:
Oh no!  It's in the nxp middleware usb sdk.  device/usb_device_dci.h.

Reply:
https://github.com/nxp-mcuxpresso/mcux-sdk-middleware-usb/pull/5

Reply:
@jzipperer-fb, the NXP SDK team has to review and integrate these changes first. We can then add these changes to Zephyr. For now, I have added a PR to revert the 2 commits to the NXP drivers.

Reply:
ok.  sounds good.

Reply:
how can i kick start that process?

Reply:
I have followed up with the SDK team via the PR you submitted. We can keep an eye on that PR.

Reply:
hey,
this was merged to main, causing build issues:
```
drivers/usb/device/usb_dc_mcux.c: In function 'usb_mcux_thread_main':
drivers/usb/device/usb_dc_mcux.c:913:22: error: 'kUSB_DeviceNotifySOF' undeclared (first use in this function); did you mean 'kUSB_DeviceNotifyError'?
  913 |                 case kUSB_DeviceNotifySOF:
      |                      ^~~~~~~~~~~~~~~~~~~~
      |                      kUSB_DeviceNotifyError
```
i believe these changes should be reverted, https://github.com/nxp-mcuxpresso/mcux-sdk-middleware-usb/pull/5 patch be merged into https://github.com/zephyrproject-rtos/hal_nxp/ and finally the manifest updated in this repo?

Reply:
ah, saw https://github.com/zephyrproject-rtos/zephyr/pull/67459 now.